README.md

```diff
@@ -1 +1,27 @@
-# OOP-School-library
\ No newline at end of file
+# OOP-School-
+
+An imaginary library that helps store books, and give access to a particuar age group and seek permission for student below a particular age.
+
```

Comment:
- [ ] [Optional] Add more details to your Readme file. This would make your project look more professional.